packages/alias/alias.spec.mjs

```diff
@@ -35,37 +34,23 @@ describe('alias', () => {
 		});
 	});
 
+	/**
+	 * This sub suite tests isn't important
+	 * But it's just to give context of where did the aliases come from
+	 * In the past they were a support of aliases in the package.json
+	 */
```

Comment:
I think this is not correct: these do matter because they confirm tsconfig support.

```suggestion
```